Source/Addendum/AdViewUtils.swift

```diff
@@ -27,19 +27,16 @@ public final class AdViewUtils: NSObject {
     @objc
     public static func findPrebidCreativeSize(_ adView: UIView, success: @escaping (CGSize) -> Void, failure: @escaping (Error) -> Void) {
         
-        let view = self.findWebView(adView) { (subView) -> Bool in
-            return isWebView(subView)
```

Comment:
I think this is still relevant. It should be isWebView irrestpective of WKWebView or UIWebView...

Reply:
In my opinion `WebView` is a common definition which covers the whole set of classes(`UIWebView`, `WKWebView`) with corresponded functionality. 

Theoretically `UIWebView` can be passed into it and success result is expected. That is why I decided to specify a concrete type(`WKWebView`) for this logic